public/html/decrypt.html

```diff
@@ -1,11 +1,12 @@
 <head>
   <meta charset="utf-8">
   <title>Babble Decryption</title>
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
```

Comment:
Use the Bootstrap CSS in the public directory?

Reply:
I used the same source as in options. We should probably change it there too then.

Reply:
I was not able to get it working.